src/scene/batching/skin-batch-instance.js

```diff
@@ -1,27 +1,26 @@
 import { SkinInstance } from '../skin-instance.js';
 
 // Class derived from SkinInstance with changes to make it suitable for batching
-function SkinBatchInstance(device, nodes, rootNode) {
-    SkinInstance.call(this);
+class SkinBatchInstance extends SkinInstance {
 
-    var numBones = nodes.length;
-    SkinInstance.prototype.init.call(this, device, numBones);
+    constructor(device, nodes, rootNode) {
 
-    this.device = device;
-    this.rootNode = rootNode;
+        super();
 
-    // Unique bones per clone
-    this.bones = nodes;
-}
+        var numBones = nodes.length;
+        SkinInstance.prototype.init.call(this, device, numBones);
```

Comment:
Does `this.init(device, numBones);` not work?

Reply:
Should be `super.init(device, numBones)`;

Reply:
changed to this.init. No need for super.init and this does not implement init.